api/models/process.go

```diff
@@ -247,32 +249,32 @@ func ListProcesses(app string) ([]*Process, error) {
 		select {
 		case ps := <-psch:
 			pss = append(pss, &ps)
-		case err := <-errch:
-			return nil, err
 		}
 	}
 
 	pending, err := ListPendingProcesses(app)
-
 	if err != nil {
-		return nil, err
+		fmt.Printf("ns=kernel at=ListProcesses state=error message=\"unable to get pending processes: %s\"\n", err)
 	}
 
 	pss = append(pss, pending...)
 
 	oneoff, err := ListOneoffProcesses(app)
 
 	if err != nil {
-		return nil, err
+		fmt.Printf("ns=kernel at=ListProcesses state=error message=\"unable to get one-off processes: %s\"\n", err)
 	}
 
 	pss = append(pss, oneoff...)
 
 	return pss, nil
 }
 
-func ListPendingProcesses(app string) ([]*Process, error) {
-	pss := make([]*Process, 0)
+// ListPendingProcesses tries to get a list of all pending processes.
+// If unable to connect to a docker daemon, or error out for another reason, it will bypass that instance and continue to other daemons.
```

Comment:
This comment isn't right. From my reading of the code ListPendingProcesses talks to CloudFormation to get all the ECS Service names for a stack, then talks to ECS to describe the services and their deployment status.

It doesn't talk to Docker or an individual instance directly.